common/scala/src/main/resources/logging.conf

```diff
@@ -1,4 +1,4 @@
 akka {
-  loglevel = "DEBUG"
+  loglevel = "INFO"
```

Comment:
isn't this changing the log level for akka itself and hence might this generate a lot more logs?